src/software/gui/standalone_simulator/widgets/standalone_simulator_draw_function_visualizer.cpp

```diff
@@ -13,6 +13,9 @@ StandaloneSimulatorDrawFunctionVisualizer::StandaloneSimulatorDrawFunctionVisual
 {
     // Let mouseMoveEvents be triggered even if a mouse button is not pressed
     this->setMouseTracking(true);
+
+    shift_clicked = false;
+    ctrl_clicked  = false;
```

Comment:
ideally, these should set in the initializer list, i.e. `DrawFunctionVisualizer(parent), shift_clicked(false), ctrl_clicked(false)`

Reply:
Did not know that about cpp! Done